src/NuGet.Core/NuGet.Common/Utility/NuGetLogCodeUtility.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.Common
+{
+    public static class NuGetLogCodeUtility
+    {
+        public static bool AreVulnerabilitiesInCodes(IEnumerable<NuGetLogCode> nugetLogCodes)
+        {
+            foreach (var code in nugetLogCodes)
+            {
+                if (IsVulnerabilityCode(code))
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        public static bool IsVulnerabilityCode(NuGetLogCode nuGetLogCode)
+        {
+            if (nuGetLogCode.Equals(NuGetLogCode.NU1901) ||
+                nuGetLogCode.Equals(NuGetLogCode.NU1902) ||
+                nuGetLogCode.Equals(NuGetLogCode.NU1903) ||
+                nuGetLogCode.Equals(NuGetLogCode.NU1904))
```

Comment:
```suggestion
            if (nuGetLogCode == NuGetLogCode.NU1901 ||
                nuGetLogCode == NuGetLogCode.NU1902 ||
                nuGetLogCode == NuGetLogCode.NU1903 ||
                nuGetLogCode == NuGetLogCode.NU1904)
```

`someEnum.Equals(anotherEnum)` causes boxing (the only `.Equals` method takes `object` as a parameter): https://sharplab.io/#v2:EYLgHgbALANALiAlgGwD4AEBMBGAsAKHQGYACLEgYQIG8CT6zTgB7Z5EgWWwAoBREgKYBKOg1r4GksgHZBAOl4BHAK4BDZAGc+cgCoB3ZkIDco+gF9TJS8RIs2nTH0EiJYy5PSyBJALw+SvLoGJq7mltakAgB2ygC2ASTiUgwA8lECMCT6zJYW+GZAA=